pkg/operator/controller/ingress/deployment.go

```diff
@@ -1113,10 +1120,18 @@ func accessLoggingForIngressController(ic *operatorv1.IngressController) *operat
 
 	switch ic.Spec.Logging.Access.Destination.Type {
 	case operatorv1.ContainerLoggingDestinationType:
+		var ContainerLoggingParameters *operatorv1.ContainerLoggingDestinationParameters
+		if ic.Spec.Logging.Access.Destination.Container == nil {
+			ContainerLoggingParameters = &operatorv1.ContainerLoggingDestinationParameters{}
+		} else {
+			ContainerLoggingParameters = &operatorv1.ContainerLoggingDestinationParameters{
+				MaxLength: ic.Spec.Logging.Access.Destination.Container.MaxLength,
+			}
+		}
 		return &operatorv1.AccessLogging{
 			Destination: operatorv1.LoggingDestination{
 				Type:      operatorv1.ContainerLoggingDestinationType,
-				Container: &operatorv1.ContainerLoggingDestinationParameters{},
+				Container: ContainerLoggingParameters,
```

Comment:
Change `ContainerLoggingParameters` to `containerLoggingParameters`.  You can simplify the logic as follows:
```suggestion
		var containerLoggingParameters operatorv1.ContainerLoggingDestinationParameters
		if ic.Spec.Logging.Access.Destination.Container != nil {
			containerLoggingParameters.MaxLength = ic.Spec.Logging.Access.Destination.Container.MaxLength
		}
		return &operatorv1.AccessLogging{
			Destination: operatorv1.LoggingDestination{
				Type:      operatorv1.ContainerLoggingDestinationType,
				Container: &containerLoggingParameters,
```